module1/homework/grayscaleImages/compression.cpp

```diff
@@ -0,0 +1,49 @@
+#include "compression.hpp"
+
+#include <cstdint>
+
+std::vector<std::pair<uint8_t, uint8_t>> compressGrayscale(const std::array<std::array<uint8_t, width>, height>& bitmap) {
+    uint8_t counter = 1;
```

Comment:
Isn't it works just with int?

Reply:
It would work with int,but since we are using uint8_t for compression(according to task requirment). We are using the same type for counter to have identical range and avoid comparing different types.